### PR TITLE
Paginate transfers page

### PR DESCRIPTION
**Pull request by exu3**

### CODE REVIEW

File: app/views/events/transfers.html.erb [hunk 1]
```diff
@@ -5,7 +5,7 @@
 <h1 class="heading">
   <span class="flex items-center">
     Transfers
-    <%= badge_for @transfers.count, class: "bg-muted" %>
+    <%= badge_for @transfers_count, class: "bg-muted" %>
```
Comment (cjdenio) [inline app/views/events/transfers.html.erb]:
```suggestion
    <%= badge_for @transfers.total_count, class: "bg-muted" %>
```

File: app/controllers/events_controller.rb [hunk 2]
```diff
@@ -295,7 +295,9 @@ def transfers
       @disbursements = @disbursements.search_name(params[:q]) if params[:q].present?
     end
 
-    @transfers = (@checks + @ach_transfers + @disbursements).sort_by { |o| o.created_at }.reverse!
+    @transfers_count = (@checks + @ach_transfers + @disbursements).count
+
```
Comment (cjdenio) [inline app/controllers/events_controller.rb]:
```suggestion
```

**Review by cjdenio — APPROVED** [review]
woo yeah.

I left a little refactor as a suggested change, but this looks great!

File: app/views/events/transfers.html.erb (hunk 1 above)
Comment (exu3) [reply app/views/events/transfers.html.erb]: Oh neat thanks, I didn't know `total_count` was a thing!

**Pull request merged**